src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/custom.py

```diff
@@ -37,3 +38,37 @@ def cli_redis_update_settings(client, resource_group_name, name, redis_configura
         existing.static_ip,
         )
     return client.create_or_update(resource_group_name, name, parameters=update_params)
+
+def cli_redis_create(client, resource_group_name, name, location, sku_name=None, # pylint:disable=too-many-arguments
```

Comment:
>sku_name=None [](start = 66, length = 13)

Why not just make sku_name not have a default so you don't have to specify required=True? Same with sku_family and sku_capacity?